frontend/src/icons/navigation/calendar-dark-gray.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="21.875" height="25" viewBox="0 0 21.875 25">
+  <path id="Icon_awesome-calendar-alt" data-name="Icon awesome-calendar-alt" d="M0,22.656A2.344,2.344,0,0,0,2.344,25H19.531a2.344,2.344,0,0,0,2.344-2.344V9.375H0Zm15.625-9.57a.588.588,0,0,1,.586-.586h1.953a.588.588,0,0,1,.586.586v1.953a.588.588,0,0,1-.586.586H16.211a.588.588,0,0,1-.586-.586Zm0,6.25a.588.588,0,0,1,.586-.586h1.953a.588.588,0,0,1,.586.586v1.953a.588.588,0,0,1-.586.586H16.211a.588.588,0,0,1-.586-.586Zm-6.25-6.25a.588.588,0,0,1,.586-.586h1.953a.588.588,0,0,1,.586.586v1.953a.588.588,0,0,1-.586.586H9.961a.588.588,0,0,1-.586-.586Zm0,6.25a.588.588,0,0,1,.586-.586h1.953a.588.588,0,0,1,.586.586v1.953a.588.588,0,0,1-.586.586H9.961a.588.588,0,0,1-.586-.586Zm-6.25-6.25a.588.588,0,0,1,.586-.586H5.664a.588.588,0,0,1,.586.586v1.953a.588.588,0,0,1-.586.586H3.711a.588.588,0,0,1-.586-.586Zm0,6.25a.588.588,0,0,1,.586-.586H5.664a.588.588,0,0,1,.586.586v1.953a.588.588,0,0,1-.586.586H3.711a.588.588,0,0,1-.586-.586ZM19.531,3.125H17.188V.781A.784.784,0,0,0,16.406,0H14.844a.784.784,0,0,0-.781.781V3.125H7.812V.781A.784.784,0,0,0,7.031,0H5.469a.784.784,0,0,0-.781.781V3.125H2.344A2.344,2.344,0,0,0,0,5.469V7.812H21.875V5.469A2.344,2.344,0,0,0,19.531,3.125Z" fill="#ccb8ad"/>
```

Comment:
Na, ezt én szúrtam el, az ikonok rosszul vannak elnevezve. Ez a sima gray ikon, a gray meg a dark gray. De ha megnézed látod is, a gray-nek kellene világosabbnak lennie, a dark graynek sötétebbnek, szóval cseréljük meg a két calendar file-t.